repos/system_upgrade/el7toel8/actors/persistentnetnames/actor.py

```diff
@@ -0,0 +1,17 @@
+import os
+import pyudev
+
+from leapp.libraries.common import *
+from leapp.actors import Actor
+from leapp.models import PCIAddress, Interface, PersistentNetNamesFacts
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
```

Comment:
There are unused imports and missing import:

```
import leapp.libraries.common.persistentnetnames
```

Similar errors are in other actors. Try to scan the code using Flake8 or pylint.